code/modules/research/mechanic/flatpack.dm

```diff
@@ -147,6 +147,13 @@
 		if((stacked.len + stacking.stacked.len + 2) >= MAX_FLATPACK_STACKS) //how many flatpacks we can in a stack (including the bases)
 			to_chat(user, "You can't stack flatpacks that high.")
 			return
+		if(user.incapacitated() || user.lying)
```

Comment:
I'm pretty sure lying checks for people resting.  And incapacitated checks for crit/death.  I copied it from the block above that was written by @clusterfack 9 months ago.